b2b-sample/guardio-insurance-sample-application/components/sections/settingsSection/idpSection/otherComponents/buttonGroupIdentityProviderDetails.js

```diff
@@ -0,0 +1,127 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Trash from "@rsuite/icons/Trash";
+import React, { useCallback, useEffect, useState } from "react";
+import { Button, IconButton, Stack, useToaster } from "rsuite";
+import ConfirmAddRemoveLoginFlowModal from "./confirmAddRemoveLoginFlowModal";
+import decodeGetApplication from "../../../../../util/apiDecode/settings/application/decodeGetApplication";
+import decodeListCurrentApplication from 
+    "../../../../../util/apiDecode/settings/application/decodeListCurrentApplication";
+import decodeDeleteIdentityProvider from
+    "../../../../../util/apiDecode/settings/identityProvider/decodeDeleteIdentityProvider";
+import { checkIfIdpIsinAuthSequence } from "../../../../../util/util/applicationUtil/applicationUtil";
+import { checkIfJSONisEmpty } from "../../../../../util/util/common/common";
+import { errorTypeDialog, successTypeDialog } from "../../../../common/dialog";
+
+/**
+ * 
+ * @param prop - session, idpDetails, fetchAllIdPs, id (idp id)
+ * @returns Add/Remove button and delete button group in an Idp
+ */
+export default function ButtonGroupIdentityProviderDetails(prop) {
+
+    const { session, idpDetails, fetchAllIdPs, id } = prop;
+
+    const toaster = useToaster();
+
+    const [ allApplications, setAllApplications ] = useState({});
+    const [ applicationDetail, setApplicationDetail ] = useState({});
+    const [ idpIsinAuthSequence, setIdpIsinAuthSequence ] = useState(null);
+    const [ openListAppicationModal, setOpenListAppicationModal ] = useState(false);
+
+    const fetchData = useCallback(async () => {
+        const res = await decodeListCurrentApplication(session);
+
+        await setAllApplications(res);
+    }, [ session ]);
+
+    const fetchApplicatioDetails = useCallback(async () => {
+        if (!checkIfJSONisEmpty(allApplications) && allApplications.totalResults !== 0) {
+            const res = await decodeGetApplication(session, allApplications.applications[0].id);
+
+            await setApplicationDetail(res);
+        }
+    }, [ session, allApplications ]);
+
+    useEffect(() => {
+        fetchData();
+    }, [ fetchData ]);
+
+    useEffect(() => {
+        fetchApplicatioDetails();
+    }, [ fetchApplicatioDetails ]);
+
+    useEffect(() => {
+        if (!checkIfJSONisEmpty(applicationDetail)) {
+            setIdpIsinAuthSequence(checkIfIdpIsinAuthSequence(applicationDetail, idpDetails));
+        }
+    }, [ idpDetails, applicationDetail ]);
+
+    const onIdpDelete = (response) => {
+        if (response) {
+            successTypeDialog(toaster, "Success", "Identity Provider Deleted Successfully");
```

Comment:
`Identity Provider is deleted successfully`

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/130d957728499e5e0e731eba0b663086f931fe95